package.json

```diff
@@ -43,7 +43,11 @@
     "federalist": "npm install && gulp build && fractal build",
     "lint": "eslint src/js/**/*.js spec/**/*.js",
     "mocha": "mocha",
+    "precommit": "lint-staged --allow-empty",
     "prepare": "gulp build",
+    "_postinstall": "husky install",
```

Comment:
when the publish process runs, it changes `postinstall` to `_postinstall`, then publish, then returns it to the initial state of `postinstall` through the `pnst` library

Reply:
@mejiaj manually ran  `postpublish`, to convert the `postinstall` command back to default state

<img width="891" alt="Screen Shot 2021-12-08 at 10 09 43 AM" src="https://user-images.githubusercontent.com/37077057/145232893-9b607bf0-618e-4e45-941b-1f2f7bc5b395.png">